client/src/components/AlertSnackbar/AlertSnackbar.tsx

```diff
@@ -12,14 +13,39 @@ const AlertSnackbar = () => {
   const { open, setOpen, alertMessage, alertSeverity } = useContext(
     AlertContext
   );
+  const [duration, setDuration] = React.useState<number>(0);
   const anchorOrigin: SnackbarOrigin = {
     vertical: "top",
     horizontal: "center",
   };
 
+  const handleClose = (event?: React.SyntheticEvent, reason?: string) => {
+    if (reason === "clickaway") {
```

Comment:
Could you please add a comment to explain why there is a special check for "clickaway"?